src/webgpu/api/validation/render_pipeline/depth_stencil_state.spec.ts

```diff
@@ -205,6 +209,37 @@ g.test('depth_write,frag_depth')
     t.doCreateRenderPipelineTest(isAsync, hasDepth, descriptor);
   });
 
+g.test('depth_bias')
+  .desc(`Depth bias parameters are only valid with triangle topologies.`)
+  .params(u =>
+    u
+      .combine('isAsync', [false, true])
+      .combine('topology', kPrimitiveTopology)
+      .beginSubcases()
+      .combine('depthBias', [undefined, 0, 1])
+      .combine('depthBiasSlopeScale', [undefined, 0, 1])
+      .combine('depthBiasClamp', [undefined, 0, 1])
+  )
+  .fn(t => {
+    const { isAsync, topology, depthBias, depthBiasSlopeScale, depthBiasClamp } = t.params;
+
+    const isTriangleTopology = topology === 'triangle-list' || topology === 'triangle-strip';
+    const hasDepthBias = !!depthBias || !!depthBiasSlopeScale || !!depthBiasClamp;
+
+    const descriptor = t.getDescriptor({
+      primitive: { topology },
+      depthStencil: {
+        format: 'depth24plus',
+        depthWriteEnabled: true,
+        depthCompare: 'less-equal',
+        depthBias,
+        depthBiasSlopeScale,
+        depthBiasClamp,
+      },
+    });
+    t.doCreateRenderPipelineTest(isAsync, !hasDepthBias || isTriangleTopology, descriptor);
```

Comment:
nit: Give `!hasDepthBias || isTriangleTopology` a variable name, it's a little hard to read when I have to guess whether it's "expect success" or "expect failure"